### PR TITLE
feat(label): add component tokens

### PR DESCRIPTION
**Pull request by jcfranco**
**Related Issue:** #7180 

## Summary

Adds the following component tokens (CSS props):

 * `--calcite-label-space-y-end`
 * `--calcite-label-text-color`

Deprecates the following:

* `--calcite-label-margin-bottom`

### CODE REVIEW

**Review by macandcheese — APPROVED** [review]
Looks good pending alignment with resolution of https://github.com/Esri/calcite-design-system/pull/8790 nomenclature discussion for `margin-bottom` / `space-block-end`

File: packages/calcite-components/src/components/alert/alert.scss [hunk 1]
```diff
@@ -4,19 +4,50 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-alert-width: Specifies the width of the component.
+ * @prop --calcite-alert-background-color: Specifies the background color of the component.
```
Comment (alisonailea) [inline packages/calcite-components/src/components/alert/alert.scss:7]: I think this file should be removed from this PR.

**Review by alisonailea — CHANGES_REQUESTED** [review]

File: packages/calcite-components/src/components/alert/alert.scss (hunk 1 above)
Comment (jcfranco) [reply packages/calcite-components/src/components/alert/alert.scss:7]: Thanks for the catch. 🎣 Fixing!

**Review by alisonailea — APPROVED** [review]

**Comment by jcfranco** [comment]
**Note:** if a different base font size was previously used, there will be a difference in spacing between elements in labels. This is because the gap is now using px and no longer rem-based.

**Pull request merged**